doc/api/stream.md

```diff
@@ -484,6 +484,16 @@ added: v11.4.0
 
 Is `true` if it is safe to call [`writable.write()`][stream-write].
 
+##### writable.writableFinished
+<!-- YAML
+added: v12.6.0
+-->
+
+* {boolean}
+
+Is `true` if all data has been flushed to the underlying system. After
+the [`'finish'`][] event has been emitted.
```

Comment:
I know you only moved this sentence and didn't add it in this PR, but it seems problematic. First, it's a sentence fragment and not a full sentence. Second, it's a bit unclear. I'm not sure if this is what is meant...:

> Is `true` if all data has been flushed to the underlying system after the `finish` event has been emitted.

...or this:

> Is `true` if all data has been flushed to the underlying system. In other words, is `true` if the `finish` event has been emitted.

So, not a blocking objection, but if you fix it, I also won't complain. :-D

Reply:
As per https://github.com/nodejs/node/pull/28007, maybe @zero1five could suggest more unambiguous wording?

Reply:
> Is `true` if all data has been flushed to the underlying system.

`flushed to the underlying system` usually Invisible to the user, and may be some repetition in this sentence [#Event: 'finish'](https://nodejs.org/dist/latest-v12.x/docs/api/stream.html#stream_event_finish).

> Is `true` if after the [`'finish'`][] event has been emitted.

Only these may be more suitable for `writableFinished` future USES and instructions, imo.

Reply:
Sorry, I still am not sure what the new wording should be(

Reply:
@vsemozhetbyt I think it would be better.

before:

> Is `true` if all data has been flushed to the underlying system. After the [`'finish'`][] event has been emitted.

after:

> Is true if after the ['finish'][] event has been emitted.

Reply:
@Trott Are you OK with this variant?

Reply:
> @Trott Are you OK with this variant?

Yes. (Maybe leave it in a separate commit for now so it's easy to rebase out in case someone in @nodejs/streams disagrees?)

Reply:
Done. Dear reviewers, please check if this is still OK for you.